content/release-10-15-0/datahub-10-15-0-bundle/10_15_0.md

```diff
@@ -0,0 +1,49 @@
+---
+weight: 40
+title: Release 10.15.0
+layout: bundle
+---
+
+Cumulocity IoT DataHub Release 10.15 includes the following improvements, limitations, and known issues:
+
+### New version of internal query engine
+
+Cumulocity IoT DataHub now uses version 22 of Dremio as its query engine. The new version comes with various enhancements and improvements including performance improvements and optimized data lake access.
+
+### Change of data lake credentials
+
+When setting up Amazon S3 or Azure Storage as data lake for Cumulocity IoT DataHub, a corresponding access secret or access key is defined. These access credentials may change over time. Prior to this release, updating the credentials required to delete and reconfigure all settings. With this release, the access credentials can be changed without deleting and reconfiguring settings. 
+
+### Additional Dremio users
+
+When setting up Cumulocity IoT DataHub, a Dremio user is created. This user can be used in applications to run SQL queries against the data lake via Dremio. With this release, additional Dremio users for running queries can be created and managed using Cumulocity IoT DataHub.
+
+### Mixed Type Coercion 
+
+Prior to DataHub 10.15, if an attribute of a collection had varying types associated, the result table would contain a mixed-type column, leading to problems with applications consuming the data over JDBC/ODBC. With 10.15 mixed-type columns are now automatically coerced to a generic type that can represent all involved types. For example, if a mixture of integer and floating point values is given, it will coerce to float. In most cases, data will be coerced to varchar/String as generic type.
```

Comment:
```suggestion
Prior to DataHub 10.15, if an attribute of a collection had varying types associated, the result table would contain a mixed-type column, leading to problems with applications consuming the data over JDBC/ODBC. With 10.15 mixed-type columns are now automatically coerced to a generic type that can represent all involved types. For example, if a mixture of integer and floating point values is given, it will coerce to float. In most cases, data will be coerced to varchar/string as generic type.
```